pages/1.11/installing/faq/index.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
This file doesn't look moved it looks new, and I'm not seeing which enterprise tags are changed.